src/slider/styled-components.js

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import {styled} from '../styles/index';
+import {StyledBarProgress} from '../progress-bar';
+
+export const Root = styled('div', props => {
+  return {
+    paddingTop: '1px',
+    position: 'relative',
+  };
+});
+
+export const Axis = styled('div', props => {
+  const {$theme} = props;
+  const {colors} = $theme;
+  return {
+    position: 'relative',
+    margin: '34px 10px 10px',
+    borderRadius: $theme.borders.useRoundedCorners ? '2px' : '0px',
+    background: colors.mono400,
+    height: '4px',
+  };
+});
+
+export const AxisRange = styled(StyledBarProgress, props => {
+  const {$max, $min, $index, $isRange, $value, $theme} = props;
+  const {colors} = $theme;
+  const fillColor = colors.primary400;
+  const emptyColor = colors.mono400;
+  let $color;
+  if ($isRange) {
+    $color = $index % 2 === 0 ? emptyColor : fillColor;
```

Comment:
How about we name this var `barColor` for example, since usage of the `$` may bring some confusion around if this color comes from props.

Reply:
ok